core/src/main/java/com/coderanch/blackjack/Card.java

```diff
@@ -76,13 +96,16 @@ Suit suit() {
         return this.suit;
     }
 
+    int points() {
+        return this.rank.points;
+    }
+
     /**
      * Compares this card to another card.
-     *
+     * <p>
```

Comment:
I like that you're using paragraphs, but use them as intended:
```html
/**
 * <p>Compares this card to another card.</p>
 * <p>Cards are first compared by rank, then by suit.</p>
 */

Reply:
Actually I think the autoformat put that in there.  Do you want me to use <p>?

Reply:
What do you mean by autoformat? Is it an IDE feature?

I want you to use `<p>` if it makes the generated Javadoc look more readable, but I want you to use it like proper semantic HTML, meaning the summary consists of two paragraphs, NOT two loose pieces of text separated by an empty paragraph.

If your IDE is not capable of producing semantic HTML for Javadoc comments, then don't use it to format Javadoc.

When in doubt, just remove all HTML from your Javadoc, and we'll fix it when we focus on making the generated Javadoc look good.